rust/frameworks.html.markerb

```diff
@@ -0,0 +1,19 @@
+---
+title: Rust Framework Guides
+layout: framework_docs_overview
+toc: false
+order: 4
+---
+
+
+<p>Each one of the language guides will take you through creating and deploying a simple application.</p>
+
+<ul class="py-4">
+  <% current_page.children.sort_by {|child| child.data[:order] || 99}.each do |page| %>
+    <li>
+      <%= link_to_page page %>
+    </li>
+  <% end %>
+</ul>
```

Comment:
You don't need this bit, since the frameworks layout automatically adds each child page as a subsection (with the heading being a link to the child page and the `objective` from the child page front matter being the content of the subsection).